intro.js

```diff
@@ -251,9 +254,6 @@
       //then, start the show
       _nextStep.call(self);
 
-      var skipButton     = targetElm.querySelector('.introjs-skipbutton'),
-          nextStepButton = targetElm.querySelector('.introjs-nextbutton');
```

Comment:
@bozdoz I assume we were not using this variables, right?

Reply:
Correct, jshint said they were unused.